lib/elixir_boilerplate_web/home/templates/index.html.eex

```diff
@@ -0,0 +1,10 @@
+<div align="center">
+  <br />
+  <img src="https://user-images.githubusercontent.com/11348/52080254-520cb580-2565-11e9-8c21-156cf0b7bcf3.png" width="500" />
+  <p><br />This repository is the stable base upon which we build our Elixir projects at Mirego.<br />We want to share it with the world so you can build awesome Elixir applications too.</p>
```

Comment:
The first `<br />` could be outside the paragraph

Reply:
No the space is not the same if the `br` is not inside a `p` tag